code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -45,7 +45,7 @@ Difficulty: Hard
 	pixel_x = -32
 	del_on_death = 1
 	crusher_loot = list(/obj/structure/closet/crate/necropolis/bubblegum/crusher)
-	loot = list(/obj/structure/closet/crate/necropolis/bubblegum)
+	loot = list(/obj/structure/closet/crate/necropolis/bubblegum, /obj/item/clothing/suit/space/hostile_environment, /obj/item/clothing/head/helmet/space/hostile_environment)
```

Comment:
it should be in the actual chest

Reply:
@KorPhaeron 
> @XDTM Dont put it in the table, put it in addition to the chest, like how killing a drake gives you a chest + armour

Reply:
put it IN the chest, not as a random chest drop!

Reply:
no really you can put it in the chest without it being part of the random pool, it can be in addition to chest drops